talek_net/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "2"
+
+services:
+
+  frontend:
+    restart: "no"
+    image: talek
+    volumes:
+      - ./:/talek_net
```

Comment:
maybe make the volume `talek_shared_mnt` or something that disambiguates from the docker network `net`?

Reply:
I changed the volume path to talek_shared